src/control/server/netdetect.c

```diff
@@ -0,0 +1,197 @@
+//

```

Comment:
(style)  trailing whitespace
(style)  do not use C99 // comments

Reply:
Wow, my editor setting on that file was spectacularly wrong!!  Please look at the content anyway while I fix the style issues.